.buildkite/fund-settlements.yml

```
@@ -192,22 +182,21 @@ steps:
     commands:
     - buildkite-agent artifact download --include-retried-jobs fund-report.txt . || echo 'UNKNOWN ERROR' > './fund-report.txt'
     - cat ./fund-report.txt
-    - echo "--------------------------------------------\n\n"
+    - echo "--------------------------------------------"
     - source ./scripts/execute-handlers.sh
     - check_command_execution_status "fund-settlement" || true
-    - |
-      claim_type=$(buildkite-agent meta-data get claim_type) || true
-      [[ -n "$$claim_type" ]] && claim_type=" [$$claim_type]"
-    - slack_feed=$(buildkite-agent meta-data get slack_feed)
+    - claim_type=$(buildkite-agent meta-data get claim_type)
+    - config_pubkey=$(buildkite-agent meta-data get config_pubkey)
     - attempts_count=$(buildkite-agent meta-data get attempts_count || echo "UNKNOWN")
     - |
       curl $SLACK_API -X POST -H 'Content-Type: application/json; charset=utf-8' \
         -H "Authorization: Bearer $$SLACK_BEARER_TOKEN" -d '{
-        "channel": "'"$$slack_feed"'",
+        "channel": "'$(buildkite-agent meta-data get slack_feed)'",
         "attachments": [{
           "color": "'"$$notification_color"'",
-          "title": "Fund Settlements '"$${notification_result}$${claim_type}"', '"$$attempts_count"' attempts",
+          "title": "Fund '"$$claim_type Settlements"' '"$${notification_result}"', '"$$attempts_count"' attempts",
```

(minor non-blocking note - consistently with other pipeline files)
```suggestion
          "title": "Fund '"$$claim_type"' Settlements '"$${notification_result}"', '"$$attempts_count"' attempts",
```